library/item/item_content_registry/src/main/java/org/quiltmc/qsl/item/content_registry/api/ItemContentRegistries.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.item.content_registry.api;
```

Comment:
That underline should be a dot, therefore, yeah, the whole package needs to change
```suggestion
package org.quiltmc.qsl.item.content.registry.api;
```